py/torch_tensorrt/dynamo/conversion/_ConverterRegistry.py

```diff
@@ -323,6 +329,18 @@ def __getitem__(
 
                 if isinstance(converters, (list, tuple)):
                     for candidate in converters:
+                        # TODO: Importing this here avoids circular import issue. One potential fix is moving this function into _ConverterRegistry file.
+                        from torch_tensorrt.dynamo.conversion.converter_utils import (
+                            dynamic_unsupported,
+                        )
+
+                        has_static_inputs = dynamic_unsupported(node)
```

Comment:
Consider renaming this function to `has_dynamic_shapes`, because the function is no longer a validator